### DISCUSSION TITLE
Links restrictions

### DISCUSSION THREAD

**Discussion by lykaodan** [Budibase/budibase#4028] [Help]
Hello! Is it possible to restrict links? Example, I have a view link on the table and it will navigate to the other page. But, the user with the "Basic" role can't proceed to the other page, and only the admin can. 
Is there any way to disable the link if you are a basic user? or a pop-up message that says "you cannot access this page" .

**Comment by mjashanks** [comment]
Yes. Your navigation is included in the `Layouts` section.

![image](https://user-images.githubusercontent.com/3524181/149550301-14c17faf-7a74-4ac8-bd41-e106b2e68596.png)

In here, you can control what links are displayed, using conditions.

![image](https://user-images.githubusercontent.com/3524181/149550493-8d55bd0c-487c-42c3-8ae6-297c93199240.png)
